packages/bpk-mixins/src/mixins/_panels.scss

```diff
@@ -30,7 +30,7 @@
 ///   }
 @mixin bpk-panel {
   display: block;
-  background-color: $bpk-panel-background-color;
+  background-color: $bpk-card-background-color;
```

Comment:
deprecating panel padding and background color tokens as they should always be the same as the card ones so we can use the same token for both